provider/README.md

```diff
@@ -75,6 +75,31 @@ Response bodies must be a `UTF-8` encoded JSON object and must minimally include
 }
 ```
 
+#### Provider HTTP Codes
```

Comment:
Probably don't need the word *Provider* here. Maybe *HTTP Response Codes*?

Reply:
Sure.